test/sasdataloader/test/utest_cansas.py

```diff
@@ -104,20 +101,18 @@ def _check_data(self, data):
         self.assertTrue(data.process[0].name == "Mantid generated CanSAS1D XML")
         self.assertTrue(data.meta_data["xmlpreprocess"] is not None)
 
-
     def _check_data_1_1(self, data):
         spectrum = data.trans_spectrum[0]
         self.assertTrue(len(spectrum.wavelength) == 138)
 
-
     def test_cansas_xml(self):
-        filename = "isis_1_1_write_test.xml"
         xmlreader = XMLreader(self.isis_1_1, self.schema_1_1)
         valid = xmlreader.validate_xml()
         xmlreader.set_processing_instructions()
         self.assertTrue(valid)
         fo = open(self.isis_1_1)
-        str = fo.read()
+        fo.read()
+        fo.close()
```

Comment:
Also, why it the code reading the file but not keeping what is read?  There are easier ways to check if a file exists.